packages/core/forms/src/index.ts

```diff
@@ -1,8 +1,8 @@
-import type { App, Plugin } from 'vue'
-import validators from './generator/utils/validators'
+import type { App } from 'vue'
+// import validators from './generator/utils/validators'
```

Comment:
nit: if some of the code isn't needed in this file (commented out) can you remove it?